apps/greeter/web/router.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Greeter.Router do
+  use Greeter.Web, :router
+
+  pipeline :browser do
+    plug :accepts, ["html"]
+    plug :fetch_session
+    plug :fetch_flash
+    plug :protect_from_forgery
+    plug :put_secure_browser_headers
+  end
+
+  pipeline :api do
+    plug :accepts, ["json"]
+  end
+
+  scope "/", Greeter do
+    pipe_through :browser # Use the default browser stack
+
+    get "/", PageController, :index
+  end
+
+  # Test conflictive routes
+  get "/starwars", Greeter.PageController, :index
+  get "/star-wars", Greeter.PageController, :index
+  get "/starwars/star-wars", Greeter.PageController, :index
+
```

Comment:
The routes above didn't generate any conflict but I couldn't test 'forward' as it needed to load Greeter.PageController which depends on Greeter.Router (recursive dependency) and I don't know why but this dependency couldn't be resolved, maybe it have something to do with not loading the Endpoint